### PR TITLE
SODEV-40075 - unbreaks the build process for node 16

### PR DESCRIPTION
**Pull request by radfahrer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,6 +66,7 @@
   },
   "homepage": "https://github.com/fpapado/react-lazy-images#readme",
   "dependencies": {
+    "eslint": "^8.29.0",
```
Comment (lisadean) [inline package.json:69]: Technically this should be a dev dependency

File: package.json [hunk 2]
```diff
@@ -73,34 +75,20 @@
...
-    "@storybook/addon-options": "4.0.0-alpha.14",
-    "@storybook/addons": "4.0.0-alpha.14",
-    "@storybook/react": "4.0.0-alpha.14",
-    "@storybook/storybook-deployer": "^2.3.0",
     "@types/react": "^18.0.18",
     "@types/react-dom": "^18.0.6",
-    "@types/storybook__addon-actions": "^3.0.3",
-    "@types/storybook__addon-info": "^3.2.3",
-    "@types/storybook__react": "^3.0.8",
-    "@typescript-eslint/parser": "^5.36.2",
```
Comment (lisadean) [inline package.json:87]: You don't need this when running `eslint`?

Comment (radfahrer) [reply package.json:87]: I was able to run eslint without it.

File: package.json (hunk 1 above)
Comment (radfahrer) [reply package.json:69]: whoops

File: package.json [hunk 3]
```diff
@@ -14,7 +14,8 @@
     "build": "npm-run-all clean bundle:prod copy-defs size",
     "bundle:prod": "microbundle build",
     "bundle:watch": "microbundle watch",
-    "copy-defs": "tsc --skipLibCheck && cp -r ts-build/*.d.ts dist/",
+    "copy-defs": "tsc --skipLibCheck --project tsconfig_base.json && cp -r ts-build/*.d.ts dist/",
+    "lint": "eslint",
     "storybook": "start-storybook -p 8080",
     "storybook:build": "npm run bundle:prod && build-storybook -c .storybook -o .out",
     "storybook:deploy": "storybook-to-ghpages --existing-output-dir=.out",
```
Comment (lisadean) [inline package.json:21]: Might as well remove these since they won't work :)

Comment (radfahrer) [reply package.json]: done

**Review by lisadean — APPROVED** [review]

**Pull request merged**